<launch>
   
	<arg name="instance_id" default="default"/>
	<param name="instance_id" value="$(arg instance_id)"/>
    <rosparam file="$(find harmoni_stt)/config/deepspeech_configuration.yaml" subst_value="True"/>
    <node pkg="harmoni_stt" type="deepspeech_service.py" name="harmoni_stt_$(arg instance_id)" output="screen"/>

</launch>
